runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/UICalculatorInputArea.java

```diff
@@ -98,7 +98,7 @@ private int getInput(JTextField field)
 	{
 		try
 		{
-			return Integer.parseInt(field.getText());
+			return Integer.parseInt(field.getText().replaceAll("\\D", ""));
```

Comment:
Actually, this should use a pre-compiled `Pattern` (ie. `private static final Pattern NON_DIGIT_MATCHER = Pattern.compile("\\D");`) and use that to replace non-digit matches.

```suggestion
			return Integer.parseInt(NON_DIGIT_MATCHER.matcher(field.getText()).replaceAll(""));
```

Reply:
I'm fine with it as it is, regardless of the subtle resolve.